src/api/basic-subscriptions.ts

```diff
@@ -114,6 +114,10 @@ export interface ClientOptions {
   allowScroll?: boolean;
   /** Skip account creation screen if requested. */
   skipAccountCreationScreen?: boolean;
+  /** Prevents links from being clicked through the background.  Clicking also
```

Comment:
This is updating public API. Are integrators expected to control this flag value? I don't think this is covered by the current design.